api/services/S3Helper.js

```diff
@@ -0,0 +1,128 @@
+const AWS = require('aws-sdk');
+const config = require('../../config');
+
+const s3Config = config.s3;
+
+const S3_DEFAULT_MAX_KEYS = 1000;
+
+const s3Client = new AWS.S3({
+  accessKeyId: s3Config.accessKeyId,
+  secretAccessKey: s3Config.secretAccessKey,
+  region: s3Config.region,
+});
+
+function listObjectsHelper(currObjects, extraS3Params = {}, opts = {}, callback) {
```

Comment:
you can also cheat the line count by just returning the callback on the same line, rather than returning undefined.